### DISCUSSION TITLE
Can we label the cells so we know what cell is dying when executing? [Python]

### DISCUSSION THREAD

**Discussion by bryanwhiting** [quarto-dev/quarto-cli#5008] [Q&A] [answered]
It's sometimes the case that I have so many cells I don't know which cell is failing.

![image](https://user-images.githubusercontent.com/18710907/228381066-500f4e58-5777-4214-be6d-3d27196a820b.png)

In r-markdown: https://bookdown.org/yihui/rmarkdown/r-code.html

```{r, chunk-label, results='hide', fig.height=4}```

I'm using Python.

**Comment by mcanouil** [comment]
I don't know if it's possible, but I would assume it is, to display the `id` (which is the notebook cell identifier, see <https://quarto.org/docs/reference/cells/cells-jupyter.html>) when available instead of the cell number in Quarto CLI standard output, as it is done in `knitr`.

@bryanwhiting There is nothing you can do on your hand here I am afraid.

<details><summary>Quarto code example.</summary>

````qmd
---
title: "Quarto- Minimal Working Example"
format: html
---

```{python}
#| label: mean
#| id: mean1
# compute the mean of a list of numbers
def mean(x):
    return sum(x) / len(x)

# assign a list of numbers to x
x = [1, 2, 3, 4, 5]

# compute the mean of x
mean(x)
```
````

</details>

<img width="1624" alt="image" src="https://user-images.githubusercontent.com/8896044/228522282-1ef0a021-4e36-4451-956e-7d0d3908339e.png">

Progress bar in `knitr` display the `label`:

<img width="1624" alt="image" src="https://user-images.githubusercontent.com/8896044/228523430-4a993df1-dff5-40a7-8ec5-2928c9a2ac9e.png">

**Comment by mcanouil** [comment, marked as answer]
FYI, added in:
- #5772

**Reply by bryanwhiting** [reply]
Thank you!!!
